### ISSUE TITLE
Error messages should have JSON format

### ISSUE THREAD

**Issue by AndiKu** [prometheus/influxdb_exporter#73]
Context:
We are using this exporter to push metrics in influxdb-format to Prometheus. We use dedicated script libary from specific software vendor on metric source. 

Issue:
This scripts pushing the metrics expect error messages as JSON and run into error if it is no JSON. 

Actual the influxdb_exporter give back simple text:
````
error parsing request: unable to parse 'my_metric': missing fields
````
To be more compatible with a normal influxdb the error messages should look like this:
````
{"error":"error parsing request: unable to parse 'my_metric': missing fields"}
````

**Comment by matthiasr** [comment]
are the InfluxDB error responses documented somewhere? or is it always `{"error":"…"}`?

**Comment by AndiKu** [comment]
I found this page in the influxDB documentation: https://docs.influxdata.com/influxdb/v1.8/troubleshooting/errors/
If I understand the code of the influxdb_exporter correctly it will only return errors (not warnings or info). So a fixed ````{"error":"..."}```` around the existing messages should be fine.

**Comment by matthiasr** [comment]
fixed by #74

**Comment by AndiKu** [comment]
Many thanks for the very quick solution. It is working fine.
